t/fixtures/01-jobs.pl

```diff
@@ -136,6 +136,27 @@
         settings   => [{key => 'DVD', value => '1'}, {key => 'DESKTOP', value => 'kde'}, {key => 'ISO_MAXSIZE', value => '4700372992'}, {key => 'ISO', value => 'openSUSE-Factory-DVD-x86_64-Build0048-Media.iso'}, {key => 'QEMUCPU', value => 'qemu64'}]
     },
     Jobs => {
```

Comment:
hm, I thought it would be a good idea to add this job as we didn't have two jobs in the same scenario with differing machine tags yet. For overall coverage I think it makes sense. Feel free to overrule me :-)